src/Data/Internal/Record.purs

```diff
@@ -0,0 +1,14 @@
+module Data.Internal.Record where
+
+-- | *Really* unsafely get a value from a record. You really shouldn't be using
+-- | this function unless you know what you're doing.
```

Comment:
Perhaps we could refine these comments a bit :smile:. The `unsafeCoerce` comment is pretty detailed actually: https://pursuit.purescript.org/packages/purescript-unsafe-coerce/3.0.0/docs/Unsafe.Coerce#v:unsafeCoerce

At least talking about the purpose / how to use them safely (such as it is) would be good.